ISL.ReIdentification.Core/Services/Coordinations/Identifications/IdentificationCoordinationService.cs

```diff
@@ -29,6 +29,21 @@ public ValueTask<AccessRequest> ProcessDelegatedAccessRequestAsync(AccessRequest
             throw new System.NotImplementedException();
 
         public ValueTask<AccessRequest> ProcessIdentificationRequestsAsync(AccessRequest accessRequest) =>
-            throw new System.NotImplementedException();
+        TryCatch(async () =>
+        {
+            ValidateAccessRequestIsNotNull(accessRequest);
+            ValidateIdentificationRequestIsNotNull(accessRequest.IdentificationRequest);
```

Comment:
Please wrap both these calls in a method on the validation file.  This prevent clutter on this method and it also helps to see what validation is done where if all that is on the validation file.

ValidateOnProcessIdentificationRequests(accessRequest)
```suggestion
            ValidateOnProcessIdentificationRequests(accessRequest)
```

Reply:
This has been updated